StyleCop.Analyzers/StyleCop.Analyzers.Test/NamingRules/SA1300UnitTests.cs

```diff
@@ -971,6 +971,39 @@ public interface IInterface
             await this.VerifyCSharpFixAsync(testCode, fixedCode).ConfigureAwait(false);
         }
 
+        [Fact]
+        [WorkItem(1935, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1935")]
+        public async Task TestUnderscoreExclusionAsync()
+        {
+            var testCode = @"public enum TestEnum
```

Comment:
This can be made into a const local variable, which would be more memory efficient. A similar fix can be applied to other variables in this file.

Reply:
➡️ This is just test code